src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/XMLMessages.properties

```diff
@@ -21,7 +21,7 @@
 # @version
 
         BadMessageKey = The error message corresponding to the message key can not be found.
-        FormatFailed = An internal error occurred while formatting the following message:\n  
+        FormatFailed = An internal error occurred while formatting the following message:\n\u0020\u0020
```

Comment:
Likely a mistake, since as you stated it is not in the format “foo:\u0020” as there is a newline before the trailing spaces.

However if intentional it should probably be `FormatFailed = An internal error occurred while formatting the following message:\n\u0020`